core/src/main/scala/org/apache/spark/executor/TaskMetrics.scala

```diff
@@ -44,42 +44,62 @@ class TaskMetrics extends Serializable {
   /**
    * Host's name the task runs on
    */
-  var hostname: String = _
-
+  private var _hostname: String = _
+  def hostname = _hostname
+  private[spark] def setHostname(value : String) = _hostname = value
```

Comment:
`Should be "value: String"` with no space (not sure why our style checks didn't catch this).